issuance/issuance.go

```diff
@@ -598,16 +598,22 @@ type IssuanceRequest struct {
 	SCTList           []ct.SignedCertificateTimestamp
 }
 
-// Issue generates a certificate from the provided issuance request and
-// signs it. Before signing the certificate with the issuer's private
-// key, it is signed using a throwaway key so that it can be linted using
-// zlint. If the linting fails, an error is returned and the certificate
-// is not signed using the issuer's key.
-func (i *Issuer) Issue(req *IssuanceRequest) ([]byte, error) {
+// And IssuanceToken represents an assertion that Issuer.Lint has generated
+// a linting certificate for a given input and run the linter over it with no
+// errors. It can be used to issue a real certificate (once).
+type IssuanceToken struct {
+	template *x509.Certificate
+	pubKey   any
+}
+
+// Lint generates a linting certificate, runs the linter over it, and if successful
+// returns both the linting certificate (which can be stored) and an IssuanceToken
+// (which can be used to issue a real certificate).
+func (i *Issuer) Lint(req *IssuanceRequest) ([]byte, *IssuanceToken, error) {
```

Comment:
It feels ever-so-slightly-odd to call this ".Lint()", when it does so much more (namely, constructing the template itself). Maybe ".Prepare()"? I dunno.

Reply:
Yeah, Prepare is Good. I've switched to that.